package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dio.js",
-  "version": "8.0.3",
+  "version": "8.1.0",
```

Comment:
Don't version in a feature commit

Reply:
Generally: commits should be isolated changes merged into master
From master, releases are cut based on stability and changes from the previous release

Reply:
You're right, that could be a potential foot-gun.